tcpclient/connection.go

```diff
@@ -34,6 +34,7 @@ func (c Connection) String() string {
 	return fmt.Sprintf("Connection %d is %s", c.Id, status)
 }
 
+<<<<<<< HEAD
```

Comment:
bad commit?

Reply:
I have the feeling you reviewed an intermediate commit... or you started reviewing very soon? :)